src/ContainerCommitModal.jsx

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import {Modal, Button} from 'react-bootstrap';
+
+class ContainerCommitModal extends React.Component {
+    constructor(props) {
+        super(props);
+        this.initialState = {
+            imageName: "",
+            tag: "",
+            author:"",
+            message: "",
+            command: this.props.containerWillCommit.Config ? this.props.containerWillCommit.Config.Cmd.join(" ") : "",
+            pause: true,
+            setonbuild: false,
+            onbuild: [""],
+            format: "oci",
+            selectedFormat: "oci",
+            onbuildDisabled: true,
+        };
+        this.state = this.initialState;
+        this.handleInputChange = this.handleInputChange.bind(this);
+        this.handleCancel = this.handleCancel.bind(this);
+        this.handleCommit = this.handleCommit.bind(this);
+        this.handleOnbuildsInputChange = this.handleOnbuildsInputChange.bind(this);
+        this.handleAddOnbuild = this.handleAddOnbuild.bind(this);
+        this.handleRemoveOnbuild = this.handleRemoveOnbuild.bind(this);
+    }
+
+    // TODO
+    handleOnbuildsInputChange(idx, evt) {
+
+    }
+
+    handleAddOnbuild() {
+        console.log(this.state.onbuild);
+        this.setState({onbuild: this.state.onbuild.concat([""])});
+    }
+
+    handleRemoveOnbuild(idx) {
+        this.setState({onbuild: this.state.onbuild.filter((bud, sidx) => idx !== sidx)});
+    }
+
+    // TODO
+    handleInputChange(event) {
+
+    }
+
+    // TODO
+    handleFormatChange(event) {
+
+    }
+
+    // TODO
+    handleCommit() {
+
+    }
+
+    handleCancel() {
+        this.props.handleCancelContainerCommitModal();
+        this.setState(this.initialState);
+    }
+
+    render() {
+        let onbuilds =
+            this.state.onbuild.map((bud, idx) => (
+                <div key={"onbuildvar" + idx} id="select-claimed-onbuildvars" className="containers-run-onbuildvarclaim containers-run-inline" >
+                    <form className="form-inline">
+                        <button type="button" className="btn btn-default fa fa-plus" onClick={this.handleAddOnbuild} />
+                        <button type="button" className="btn btn-default pficon-close" onClick={() => this.handleRemoveOnbuild(idx)} />
+                        <div className="form-group">
+                            <input type="text" name="onbuildvar_key" className="form-control" onChange={(evt) => this.handleOnbuildsInputChange(idx, evt)} />
+                        </div>
+                    </form>
+                </div>
+            ));
+
+        let commitContent =
+            <div>
+                <table className="form-table-ct">
+                    <tbody>
+                        <tr>
+                            <td align="right"><label className="control-label" translatable="yes">Container Name</label></td>
```

Comment:
The `translatable=` tag only works in static .html files, not in React. You need to use `<label>{_("Container Name"}}</label>`.